packages/desktop/components/popups/AddressHistoryPopup.svelte

```diff
@@ -26,7 +33,12 @@
     <Text type={TextType.h3} fontWeight={FontWeight.semibold} lineHeight="6">
         {localize('popups.addressHistory.title')}
     </Text>
-    <Text fontSize="15" color="gray-700" classes="text-left">{localize('popups.addressHistory.disclaimer')}</Text>
+    <div class="flex w-full items-center justify-between">
```

Comment:
I would rather do a big CTA button at the bottom of the popup like we have in order places in the app 
![image](https://github.com/iotaledger/firefly/assets/3624944/56883d6d-0663-4bdf-820a-85e113e52022)

Reply:
I've opted for the small button because this PR: https://github.com/iotaledger/firefly/pull/7750 also adds a Search button to the Popup, but we can add a Copy Button next to the Search :+1: